dev/breeze/src/airflow_breeze/utils/console.py

```diff
@@ -79,3 +79,15 @@ def get_console() -> Console:
         theme=get_theme(),
         record=True if recording_file else False,
     )
+
+
+@lru_cache(maxsize=None)
+def get_stderr_console() -> Console:
+    return Console(
+        force_terminal=True,
+        color_system="standard",
+        stderr=True,
+        width=180 if not recording_width else int(recording_width),
```

Comment:
This is probably a larger discussion (so feel free to ignore), but doesn't it make sense to set this number to match the [flake8 max line length](https://github.com/apache/airflow/blob/c5878315f3df0854a0902d91668cc369e7466405/.flake8#L2), the [editor configuration](https://github.com/apache/airflow/blob/c5878315f3df0854a0902d91668cc369e7466405/.editorconfig#L29), the [`isort` line length](https://github.com/apache/airflow/blob/c5878315f3df0854a0902d91668cc369e7466405/setup.cfg#L208), and the [`black` line length limit](https://github.com/apache/airflow/blob/c5878315f3df0854a0902d91668cc369e7466405/pyproject.toml#L18) of 110?

Reply:
I think there are quite different "limits" applied  to console output and source code to be honest.

This is the lenght of output not the lines of code, so this is more of a "modern terminal" length - which is way more than 110, I think. And this is more of a "max" width. Rich is generally able to support shorter term width (I think) so this is more of "if you have terminal wider than 180 chars, this is where it will wrap).

There are other reasons why we have 110 length on source code (or at least I think there is). Source code is rarely printed in terminal - more often in an IDE and there are I think two reasons why we want to limit it to 110 (rather than something way bigger) to enable TWO source code windows in one IDE (left/right) without wrapping/moving the code out of the window. This is the settings any `merge` tool display two windows and being able to see both at modern moniitor (which is certainly capable of handling 110 chars per line) is important.

Of course there is no "universal font size" and we can have a lot of debate if 120 or 110 or 80 or 160 is enough so this is more of a "rule of thumb" than "hard-data". 

But another reason to limit the source code length is acccessibility. Originally i used to limit the line lenght to 80 - not because of terminal lenght but because I worked in the same team  with Peter Lundblad at Google (he is a blind programmer, Subversion contributor and he had 80 characters Braille line reader to read the source code). Nowadays 80 chars is a bit too limiting. But It kind of sticked to me that not everyone has the same capabilities as I to read the small font (and the older I am, the more i appreciate it as well).  The 110 (after 15 year and multi-line braille readers since then) seems good compromise - as 80 chars is really low for modern programming.

But I think this limit is not that important for console output (which can wrap without an issue). I think for console output, this is just a limit to make sure someone who writes the code does not take advantage of having bigger screen than most other users, because generally the output will wrap nicely with rich anyway.

So 180 is really the "please do not use your huge screen to test how it will look ilke for most other people".